localMatchApp/headers/Login.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * \class Login
+ *
+ * \brief Create the connection with the database and the core.
+ *
+ * This class check out the format of the password and check if the
```

Comment:
Remove all "This class" comments in your classes

Reply:
Done.